otter/convergence/gathering.py

```diff
@@ -205,23 +205,28 @@ def _lb_configs_from_metadata(server):
 
     # throw away any value that is malformed
 
-    def parse_config(config, key):
-        if config.get('type') != 'RackConnectV3':
-            lbid = k[len(_key_prefix):]
+    def parse_config(lb_type, lb_id, configs):
```

Comment:
No, that's fine, it's non-blocking; we already have an RCv3 convergence ticket. However, we should link this conversation there; I'll go do that.